python/lsst/validate/drp/report_performance.py

```diff
@@ -32,17 +32,14 @@ def run(validation_drp_report_filenames, output_file,
     """
     Parameters
     ---
-    validation_drp_report_filenames : [] or str, filepaths for JSON files.
-    output_file : str, filepath of output RST file.
-    srd_level : str, SRD level to quote.  One of ['design', 'minimum', 'stretch']
-    release_specs_file : str [optional], filepath of metrics YAML file
-      While the JSON file itself will store the metrics the was calculated with
-        one may wish to compare against an external set of specifications.
-      E.g., ${VALIDATE_DRP_DIR}/etc/release_specs.yaml
-        contains the target levels for each fiscal year through to and including
-        operational readiness review (ORR).  This file is essentially the same
-        as ${VALIDATE_DRP_DIR}/etc/metrics.yaml, but defines levels in terms
-        of fiscal year and ORR instead of design/minimum/stretch.
+    validation_drp_report_filenames : [] or str
```

Comment:
`list or str`

Reply:
Or is it `list of str`?

Reply:
Are you saying I should use the word list?

Reply:
Yep